src/Helpers/Portfolio/PortfolioHelper.js

```diff
@@ -59,3 +59,16 @@ export async function updatePortfolio(portfolioData) {
 export async function removePortfolio(portfolioId) {
   await userApi.destroyPortfolio(portfolioId);
 }
+
+export async function removePortfolioItem(portfolioItemId) {
+  return userApi.destroyPortfolioItem(portfolioItemId).then(data => {
+    console.log('Remove Portfolio Item Called successfully.');
+    return data;
+  });
+}
+
+export async function removePortfolioItems(portfolioItemIds) {
+  for (const itemId of portfolioItemIds) {
```

Comment:
This should be probably Promise.all. We want to wait until all the requests are successfully completed.
 You have to prepare array of promises and resolve them after they are all completed. Code should look something like this:

```javascript
export function removePortfolioItems(portfolioItemIds) {
  return Promise.all(portfolioItemIds.map(async itemId =>  await removePortfolioItem(itemId)))
}

```

Reply:
It waits for individual requests, this will wait for everything.